modules/ROOT/pages/_partials/commons/common-document.adoc

```diff
@@ -263,7 +263,7 @@ ifdef::is-android[]
 [NOTE]
 --
 Care should be taken when storing and recovering data in a document or converting that document to JSON and back. +
-Data encoding (Fleece) can result in `Long` values being converted to `Float` instead of `Double`. +
+Data encoding (Fleece) can result in `Long` values being converted to `Integers`, and `Double` values to `Float`. +
```

Comment:
Should this be just `Integer`?

```suggestion
Data encoding (Fleece) can result in `Long` values being converted to `Integer`, and `Double` values to `Float`. +
```

Reply:
You're correct. Will leave tab open and this comment unresolved to fix at a later date